src/service/calculator/calculator_repository.py

```diff
@@ -51,12 +51,49 @@ def get_metric_option(
             "alias": f"{alias}",
         }
 
-    def get_actuals_budget_options(self, year: int) -> list:
+    def get_actuals_options(self, year: int) -> list:
         return [
             self.get_metric_option("Actuals", "Revenue", "actuals_revenue", year),
             self.get_metric_option("Actuals", "Ebitda", "actuals_ebitda", year),
+            self.get_metric_option(
+                "Actuals", "Cost of good", "actuals_cost_of_goods", year
+            ),
+            self.get_metric_option(
+                "Actuals", "Sales & marketing", "actuals_sales_marketing_cost", year
+            ),
+            self.get_metric_option(
+                "Actuals",
+                "General & administration",
+                "actuals_general_admin_cost",
+                year,
+            ),
+            self.get_metric_option(
+                "Actuals",
+                "Research & development",
+                "actuals_research_development_cost",
+                year,
+            ),
+        ]
+
+    def get_actual_budget_options(self, year: int) -> list:
```

Comment:
Can you change the name? This functions seems to be only for budget

Reply:
this was called **get_budget_options**